backend/src/graphql/resolver/TransactionLinkResolver.ts

```diff
@@ -466,40 +505,80 @@ export class TransactionLinkResolver {
     @Arg('validUntil', { nullable: true }) validUntil?: string,
     @Arg('recipientAlias', { nullable: true }) recipientAlias?: string,
   ): Promise<boolean> {
-    const logger = createLogger()
-    logger.addContext('code', code.substring(0, 6))
-    logger.debug('TransactionLinkResolver.disburseTransactionLink... args=', {
-      senderGradidoId,
-      senderCommunityUuid,
-      recipientCommunityUuid,
-      recipientCommunityName,
-      recipientGradidoId,
-      recipientFirstName,
-      code,
-      amount,
-      memo,
-      validUntil,
+    const handshakeID = randombytes_random().toString()
+    const methodLogger = createLogger(`disburseTransactionLink`)
+    methodLogger.addContext('handshakeID', handshakeID)
+    if(methodLogger.isDebugEnabled()) {
+      methodLogger.debug('args=', {
+        senderGradidoId,
+        senderCommunityUuid,
+        recipientCommunityUuid,
+        recipientCommunityName,
+        recipientGradidoId,
+        recipientFirstName,
+        code,
+        amount,
+        memo,
+        validUntil,
       recipientAlias,
-    })
-    const disburseJwt = await this.createDisburseJwt(
-      senderCommunityUuid,
-      senderGradidoId,
-      recipientCommunityUuid,
-      recipientCommunityName,
-      recipientGradidoId,
-      recipientFirstName,
-      code,
-      amount,
-      memo,
-      validUntil ?? '',
-      recipientAlias ?? '',
-    )
-    try {
-      logger.debug('TransactionLinkResolver.disburseTransactionLink... disburseJwt=', disburseJwt)
-      // now send the disburseJwt to the sender community to invoke a x-community-tx to disbures the redeemLink
-      // await sendDisburseJwtToSenderCommunity(context, disburseJwt)
-    } catch (e) {
-      throw new LogError('Disburse JWT was not sent successfully', e)
+      })
+    }
+   const senderCom = await DbCommunity.findOneBy({ communityUuid: senderCommunityUuid })
```

Comment:
you could also use getCommunityByUuid from database here.

Reply:
done